ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml

```diff
@@ -65,6 +70,18 @@ presubmits:
     labels:
       master: ci.openshift.redhat.com
     name: test_pull_request_openshift_ansible_logging_37
+    optional: true
     rerun_command: /test logging
     run_if_changed: ^roles/openshift_logging.*|playbooks/.+/openshift-cluster/openshift.logging.*
     trigger: ((?m)^/test( all| logging),?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - release-3.7
+    context: ci/openshift-jenkins/tox
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_openshift_ansible_tox_310
```

Comment:
nitpick: name update?

Reply:
that's what the jenkins job is named and I'm not changing that

Reply:
it just needs to be optional on the branches where protection is off

Reply:
Right, makes sense